README.md

```diff
@@ -143,57 +136,30 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  var _amplifyConfigured = false;
-
   @override
   void initState() {
     super.initState();
     _configureAmplify();
   }
 
-  Future<void> _configureAmplify() async {
-    // Add Pinpoint and Cognito Plugins
-    await Amplify.addPlugin(AmplifyAuthCognito());
-    await Amplify.addPlugin(AmplifyAnalyticsPinpoint());
-
-    // Once Plugins are added, configure Amplify
-    await Amplify.configure(amplifyconfig);
-    setState(() {
-      _amplifyConfigured = true;
-    });
-  }
-
-  // Send an event to Pinpoint
-  Future<void> _recordEvent() async {
-    final event = AnalyticsEvent('test');
-    event.customProperties.addBoolProperty('boolKey', true);
-    event.customProperties.addDoubleProperty('doubleKey', 10);
-    event.customProperties.addIntProperty('intKey', 10);
-    event.customProperties.addStringProperty('stringKey', 'stringValue');
-    await Amplify.Analytics.recordEvent(event: event);
+  void _configureAmplify() async {
```

Comment:
```suggestion
  Future<void> _configureAmplify() async {
```

Reply:
I would suggest we move away from the `StatefulWidget` approach to configuration and to directly configuring in `main`. I think this will save a lot of headache for users who are new to Flutter and state management.

```dart
Future<void> main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await _configureAmplify();
  runApp(const MyApp());
}

Future<void> _configureAmplify() async {
  try {
    await Amplify.addPlugin(AmplifyAuthCognito());
    await Amplify.configure(amplifyconfig);
    safePrint('Successfully configured');
  } on Exception catch (e) {
    safePrint('Error configuring Amplify: $e');
  }
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return Authenticator(
      child: MaterialApp(
        builder: Authenticator.builder(),
        home: const Scaffold(
          body: Center(
            child: Text('You are logged in!'),
          ),
        ),
      ),
    );
  }
}
```

Reply:
Correct me if I am mistaken, but I think you will see a white screen for about a second with this approach as configuration can take a while.

Reply:
Oh yeah... okay you're right. With Authenticator, the `StatefulWidget` approach is better.